src/core/lib/support/manual_constructor.h

```diff
@@ -22,12 +22,137 @@
 // manually construct a region of memory with some type
 
 #include <stddef.h>
+#include <stdlib.h>
 #include <new>
 #include <type_traits>
 #include <utility>
 
 namespace grpc_core {
 
+#define ASSERT(x) \
```

Comment:
delete this, use GPR_ASSERT

(while we were in godbolt.org, it was convenient to have something else)

Reply:
Done